test/unit/rules/element-attributes-order-test.js

```diff
@@ -0,0 +1,108 @@
+import generateRuleTests from '../../helpers/rule-test-harness.js';
+
+generateRuleTests({
```

Comment:
Can you add `error` test cases to ensure the rule does not allow invalid configuration? 

Example: https://github.com/ember-template-lint/ember-template-lint/blob/149a28067ec4db022397c63ef489250ece1db825/test/unit/rules/table-groups-test.js#L867